source/server/server.cc

```diff
@@ -205,7 +199,7 @@ void InstanceImpl::initialize(Options& options, TestHooks& hooks,
   // GuardDog (deadlock detection) object and thread setup before workers are
   // started and before our own run() loop runs.
   guard_dog_.reset(
-      new Server::GuardDogImpl(*admin_scope_, *config_, ProdMonotonicTimeSource::instance_));
+      new Server::GuardDogImpl(stats_store_, *config_, ProdMonotonicTimeSource::instance_));
```

Comment:
@dnoe FYI I think this was a bug. The watchdog stats should sit off of the root stats tree vs. the admin listener.